lab-katherine/lib/read.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+let x = module.exports = {}
+const fs = require('fs')
+
+x.readThing = (callback) =>{
+  let first, second, third;
+
+  fs.readFile(`${__dirname}/../assets/one.txt`, (err, data) => {
+    // if (err) console.error(err);
+    first = data.toString('hex', 0, 8)
+
```

Comment:
Part of the requirements for the lab are to log each file as we process them to ensure they are being read in the correct order.